.github/workflows/ci.yml

```diff
@@ -0,0 +1,19 @@
+name: C/C++ CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: ubuntu-latest
+    runs-on: ubuntu-latest
+    container: devkitpro/devkitppc:latest
+ 
+    steps:
+    - uses: actions/checkout@v3.5.2
+
+    - name: Build
+      run: |
+        make -j2
```

Comment:
-j2 should be replaced with $(nproc) as I did here https://github.com/switchbrew/libnx/pull/652/commits/d6e38f2b41807b100c290717efa9b4d5e443fe3d

Runners currently have up to 4 cores but I've still seen 2 core runners recently and of course they may up the cores in the future.